chainer/functions/connection/n_step_lstm.py

```diff
@@ -329,6 +329,86 @@ def _stack_weight(ws):
 def n_step_lstm(
         n_layers, dropout_ratio, hx, cx, ws, bs, xs, seed=1337, train=True,
         use_cudnn=True):
+    """Stacked Long Short-Term Memory function for sequence inputs.
+
+    This function calculates stacked LSTM with sequences. This function gets
+    an initial hidden state :math:`h_0`, an initial cell state :math:`c_0`,
+    an input sequence :math:`x`, weight matrices :math:`W`, and bias vectors
+    :math:`b`.
+    This function calculates hidden states :math:`h_t` and :math:`c_t` for each
+    time :math:`t` from input :math:`x_t`.
+
+    .. math::
+
+       i_t = \sigma(W_0 x_t + W_4 h_{t-1} + b_0 + b_4)
+       f_t = \sigma(W_1 x_t + W_5 h_{t-1} + b_1 + b_5)
+       o_t = \sigma(W_2 x_t + W_6 h_{t-1} + b_2 + b_6)
+       a_t = \tanh(W_3 x_t + W_7 h_{t-1} + b_3 + b_7)
+       c_t = f_t \dot c_{t-1} + i_t \dot a_t
+       h_t = o_t \dot \tanh(c_t)
+
+    As the function accepts a sequence, it calculates :math:`h_t` for all
+    :math:`t` with one call. Eight weight matrices and eight bias vectors are
+    required for each layers. So, when :math:`S` layers exists, you need to
+    prepare :math:`8S` weigth matrices and :math:`8S` bias vectors.
+
+    If the number of layers ``n_layers`` is greather than :math:`1`, input
```

Comment:
Yes. I mean input of k-th layer is output of k-1-th layer, and input of 1st layer is the input layer.